docs/components/stepper.md

```diff
@@ -0,0 +1,197 @@
+
+<div>
+  <KStepper :steps="[
+      { label: 'A completed step', state: 'completed' },
+      { label: 'End' }
+    ]"
+  />
+</div>
+
+```js
+[
+  { label: 'A completed step', state: 'completed' },
+  { label: 'End' }
+]
+```
+
+Both `pending` and `error` states will bold the label, because these 2 states indicate the "current" step. `completed` indicates past steps, while `default` indicates future steps.
+
+<div>
+  <KStepper :steps="[
+      { label: 'A pending step', state: 'pending' },
+      { label: 'End' }
+    ]"
+  />
+</div>
+
+```js
+[
+  { label: 'A pending step', state: 'pending' },
+  { label: 'End' }
+]
+```
+
+<div>
+  <KStepper :steps="[
+      { label: 'An erroneous step', state: 'error' },
+      { label: 'End' }
+    ]"
+  />
+</div>
+
+```js
+[
+  { label: 'An erroneous step', state: 'error' },
+  { label: 'End' }
+]
+```
+
+The last step will never have a following divider.
+
+<div>
+  <KStepper :steps="[
+      { label: 'A default step' }
+    ]"
+  />
+</div>
+
+```js
+[{ label: 'A default step' }]
+```
+
+### maxLabelWidth
+
+The width of step labels (default is `170px`). We support numbers (will be converted to `px`), `auto`, and percentages (e.g. `25%`) for values.
+
+<KStepper :steps="longSteps" max-label-width="100" />
+
+```html
+<KStepper :steps="steps" max-label-width="100" />
+```
+## Theming
+
+| Variable                      | Purpose                                     |
+| :---------------------        | :-------------------------------            |
+| `KStepSize`                   | Height and width of step state icon         |
+| `KStepBackgroundColor`        | Background color of the step state icon     |
+| `KStepDividerColorCompleted`  | Color of divider for completed steps        |
+| `KStepDividerColorDefault`    | Color of divider for non-completed steps    |
+| `KStepCompletedColor`         | Primary color of completed state icon       |
+| `KStepCompletedSecondaryColor | Secondary color of completed state icon     |
+| `KStepDefaultColor`           | Primary color of default state icon         |
+| `KStepErrorColor`             | Primary color of error state icon           |
+| `KStepErrorSecondaryColor`    | Secondary color of error state icon         |
+| `KStepPendingColor`           | Primary color for pending state icon        |
+
+
+An example of theming the danger badge:
+
+> Note: We are scoping the overrides to a wrapper in this example
+
+<div class="k-stepper-wrapper">
+  <KStepper :steps="stepTypes" />
+</div>
+
+```html
+<template>
+  <div class="k-stepper-wrapper">
+    <KStepper :steps="steps" />
+  </div>
+</template>
+
+<style lang="scss">
+.k-stepper-wrapper {
+  --KStepSize: 40px;
+  --KStepDividerColorCompleted: var(--purple-300);
+  --KStepDividerColorDefault: var(--purple-100);
+  --KStepCompletedColor: var(--purple-300);
+  --KStepDefaultColor: var(--purple-100);
+  --KStepErrorColor: var(--steel-400);
+  --KStepPendingColor: var(--yellow-400);
+}
+</style>
+```
```

Comment:
The Theming section should not exist; we definitely do not want this level of customization (and they don't exist on the `main` branch)